aixplain/factories/corpus_factory.py

```diff
@@ -90,6 +90,11 @@ def __from_response(cls, response: Dict) -> Corpus:
         except:
             license = None
 
+        try:
+            length = int(response["segmentsCount"])
+        except:
```

Comment:
Correct to: `except Exception:`
Is segment count a necessary parameter to be returned in the response? How about logging the contents of the response if segmentsCount is not retrieved?